resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -686,22 +722,21 @@ addOreDictionaryTooltips=true
   </recipeGroup>
 
 
-  <recipeGroup name="Applied Energistics">
-
+  <recipeGroup name="Applied Energistics 2">
 
     <recipe name="ChargedCertusQuartzOre" energyCost="2400">
       <input>
-        <itemStack  modID="appliedenergistics2" itemName="tile.OreQuartzCharged"  />
+        <itemStack modID="appliedenergistics2" itemName="tile.OreQuartzCharged" />
       </input>
       <output>
-        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1"/>
+        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1" />
```

Comment:
When you break the ore you can get up to 4 crystals I thought. So who would ever use this recipe?